### PR TITLE
Fixing publish builder so one malformed file doesn't break everyone

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: runtime_builders/publish_builders.py [hunk 1]
```diff
@@ -55,18 +55,25 @@ def main():
...
+            latest = config['latest']
+            project_name = config['project']
+
+            parts = os.path.splitext(latest)
+            prefix = builder_util.RUNTIME_BUCKET_PREFIX + project_name + '-'
+            latest_file = parts[0].replace(prefix, '')
+            file_name = '{0}.version'.format(project_name)
+            full_path = builder_util.RUNTIME_BUCKET_PREFIX + file_name
+            builder_util.write_to_gcs(full_path, latest_file)
+        except ValueError as ve:
```
Comment (dlorenc) [inline runtime_builders/publish_builders.py]: What would throw the value error and the keyerror? Could we narrow the try/catch?

Comment (sharifelgamal) [reply runtime_builders/publish_builders.py]: I'm not sure about the ValueError, I'll like take that out. The KeyError is for 

```python
latest = config['latest']
project_name = config['project']
```

Comment (dlorenc) [reply runtime_builders/publish_builders.py]: I'd replace the KeyError with an 'if 'latest' not in config or 'project' not in config'

Comment (sharifelgamal) [reply runtime_builders/publish_builders.py]: PTAL

**Pull request merged**